src/weapon/w_020.c

```diff
@@ -125,7 +125,7 @@ void EntityWeaponAttack(Entity* self) {
     case 2:
         if (((self->animFrameIdx % 8) == 5) && (self->animFrameDuration == 1)) {
             g_api.func_80134714(
-                0x6A9, D_90000_8017AB44[self->ext.karmacoin.unk84], 0);
+                SFX_COLLECT_GOLD, D_90000_8017AB44[self->ext.karmacoin.unk84], 0);
```

Comment:
I think this is the only lint issue:

```diff
-            g_api.func_80134714(
-                SFX_COLLECT_GOLD, D_90000_8017AB44[self->ext.karmacoin.unk84], 0);
+            g_api.func_80134714(SFX_COLLECT_GOLD,
+                                D_90000_8017AB44[self->ext.karmacoin.unk84], 0);
```

Reply:
Do you know what's exactly causing lint to fail from this?  I tried to pay close attention to the whitespace and tabs when editing in NotePad++ but it still errors out.   Currently running the commands now in the codespace.

Reply:
I think `make format` calls the clang formatter under the hood. I don't see a config file anywhere so I think it's using default settings.